src/components/ebay-window-notice/examples/02-window-notice-fill/template.marko

```diff
@@ -0,0 +1,17 @@
+<style>
+    .demo2-container {
+        max-width: 320px;
+        height: 500px;
+        padding: 8px;
+        background: #fff;
+        border: 1px solid #eee;
+    }
+</style>
+
+<div class="demo2-container">
+    <ebay-window-notice type="window" a11y-text="Success!" fill-window>
```

Comment:
Shouldn't need `type` attr here.

Reply:
Also I see a `fill-window` attribute here and in the `marko.json`. It looks like it got renamed or removed?